.github/policies/close-stale-issues.yml

```diff
@@ -0,0 +1,55 @@
+id: close-stale-issues
```

Comment:
I'd just add a comment with a link to the docs, since this bot system is fairly new.

```suggestion
# Resource Management policy docs: https://eng.ms/docs/more/github-inside-microsoft/policies/resource-management

id: close-stale-issues
```